builder/remotecontext/git/gitutils.go

```diff
@@ -70,7 +70,7 @@ func fetchArgs(remoteURL *url.URL, ref string) []string {
 	shallow := true
 
 	if strings.HasPrefix(remoteURL.Scheme, "http") {
-		res, err := http.Head(fmt.Sprintf("%s/info/refs?service=git-upload-pack", remoteURL))
+		res, err := http.Get(fmt.Sprintf("%s/info/refs?service=git-upload-pack", remoteURL))
```

Comment:
I think a common approach is to perform a `HEAD` request, then fallback to `GET` if not supported by the server, not sure if that works for this, but would avoid special-casing GitHub